.github/workflows/chromatic.yml

```
@@ -0,0 +1,31 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: "Chromatic"
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: yarn
+      - name: Publish to Chromatic
+        if: github.ref != 'refs/heads/main'
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
+          # Should set this up from a secret: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          projectToken: adb814d4adb6
+      - name: Publish to Chromatic and auto accept changes
+        if: github.ref == 'refs/heads/main'
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
+          projectToken: adb814d4adb6
+          autoAcceptChanges: true
```

If `autoAcceptChanges` is the only difference in these two steps we could combine them:

```suggestion
      - name: Publish to Chromatic and auto accept changes
        if: 
        uses: chromaui/action@v1
        with:
          token: ${{ secrets.CI_GITHUB_TOKEN }}
          projectToken: adb814d4adb6
          # Auto accept changes for main branch commits.
          autoAcceptChanges: ${{ github.ref == 'refs/heads/main' }}
```